client/src/components/Events/SetResponse.js

```diff
@@ -46,11 +48,16 @@ class SetResponse extends Component {
     if (!this.state.dstToScene && this.props.event) {
       this.geoWatch = navigator.geolocation.getCurrentPosition(
         (position) => {
+          this.setState({
+            userLat: position.coords.latitude,
+            userLong: position.coords.longitude,
+          });
           const primary = this.props.event.eventLocations.find(location => location.name === 'scene');
           distance.get(
             {
               origin: `${position.coords.latitude}, ${position.coords.longitude}`,
               destination: `${primary.locationLatitude}, ${primary.locationLongitude}`,
+              apiKey: 'AIzaSyCsYLUZTuw9bEjK-Ps4W42HiwvF7gE8rgw',
```

Comment:
please move this to a const at the top of the file so it's not hiding amongst the logic